features/step_definitions/voucher_steps.rb

```diff
@@ -25,13 +25,20 @@
   sub_vouchers[0, num_total.to_i - num_free.to_i].each { |v| v.reserve_for(show.showdates.first, Customer.boxoffice_daemon) }
 end
 
-Given /^customer "(.*) (.*)" has (\d+) subscriber reservations for (.*)$/ do |first,last,num,date|
+Given /^customer "(.*) (.*)" has (\d+) non-cancelable subscriber reservations for (.*)$/ do |first,last,num,date|
   @customer = find_or_create_customer first,last
   @showdate = Showdate.find_by_thedate! Time.parse(date) unless date =~ /that performance/
   sub_vouchers = setup_subscriber_tickets(@customer, @showdate.show, num)
   sub_vouchers.each { |v| v.reserve_for(@showdate, Customer.boxoffice_daemon) }
 end
 
+Given /^customer "(.*) (.*)" has (\d+) cancelable subscriber reservations for (.*)$/ do |first,last,num,date|
```

Comment:
it would be much better to unify this step def with the one at line 28, using an optional capture group for the "non-" of "non-cancelable" and then passing the final argument in line 31 accordingly